README.md

```diff
@@ -2,7 +2,9 @@
 
 TypeScript seed project. Use it to scaffold a TS project with all the tooling need to *properly* (opinated) set up a project.
 
-[![Build Status](https://travis-ci.org/metadevpro/ts-project-seed.svg?branch=master)](https://travis-ci.org/metadevpro/ts-project-seed)
+[![Travis](https://img.shields.io/travis/omachala/ts-project-seed.svg)](https://travis-ci.org/omachala/ts-project-seed)
```

Comment:
Hi @omachala  
Review URLs to fix this pull-request. Should be pointing to the base project: in this case `metadevpro/ts-project-seed`.
This should be changed to local ones when instantiating the project-seed.